lib/messages.js

```diff
@@ -39,6 +39,8 @@ const RESTORE_STASH_EXAMPLE = `  Any lost modifications can be restored from a g
     > git stash apply --index stash@{0}
 `
 
+const CONFIG_STDIN_ERROR = 'Error: Could not read config from stdin.'
```

Comment:
Sorry for nitpicking, but a more conventional message would look something like:

```suggestion
const CONFIG_STDIN_ERROR = chalk.redBright(`${error} Could not read config from stdin!`)
```

Reply:
I was matching the style of the config parsing error messages, which do not use color it seems.

Reply:
Ah well, that's something we should improve as well. 🤷